### ISSUE TITLE
LVM volume creation fails on ubuntu-20.04

### ISSUE THREAD

**Issue by tomyun** [easimon/maximize-build-space#2]
GitHub Actions recently started upgrading Linux virtual environment from ubuntu-18.04 to ubuntu-20.04 (actions/virtual-environments#1816) that seemingly breaks maximize-build-space.

```
Run ***/maximize-build-space
  with:
    remove-dotnet: true
    remove-android: true
    root-reserve-mb: 1024
    swap-size-mb: 4096
    pv-loop-path: /pv.img
    remove-haskell: false
Memory and swap:
              total        used        free      shared  buff/cache   available
Mem:        7121276      529448     4858396        6580     1733432     6279844
Swap:       4194300           0     4194300

NAME          TYPE SIZE USED PRIO
/mnt/swapfile file   4G   0B   -2

Available storage:
Filesystem      Size  Used Avail Use% Mounted on
/dev/root        84G   60G   24G  72% /
devtmpfs        3.4G     0  3.4G   0% /dev
tmpfs           3.4G   12K  3.4G   1% /dev/shm
tmpfs           696M  1.1M  695M   1% /run
tmpfs           5.0M     0  5.0M   0% /run/lock
tmpfs           3.4G     0  3.4G   0% /sys/fs/cgroup
/dev/sda15      105M  9.2M   96M   9% /boot/efi
/dev/loop0      136M  136M     0 100% /snap/chromium/1466
/dev/loop2       56M   56M     0 100% /snap/core18/1944
/dev/loop1      138M  138M     0 100% /snap/chromium/1479
/dev/loop3       56M   56M     0 100% /snap/core18/1988
/dev/loop4      163M  163M     0 100% /snap/gnome-3-28-1804/145
/dev/loop5       65M   65M     0 100% /snap/gtk-common-themes/1514
/dev/loop6       70M   70M     0 100% /snap/lxd/19188
/dev/loop7       32M   32M     0 100% /snap/snapd/10707
/dev/loop8       32M   32M     0 100% /snap/snapd/11036
/dev/loop9       70M   70M     0 100% /snap/lxd/19032
/dev/sdb1        14G  4.1G  9.0G  32% /mnt

Arguments:

  Root reserve: 1024 MiB
  Swap space:   4096 MiB
  Mount path:   /home/runner/work/Cropbox.jl/Cropbox.jl
  PV loop path: /pv.img
  Removing:     dotnet android 

Removing unwanted software... 
... done
Unmounting swap and temp disk.
Creating LVM Volume.
  Physical volume "/dev/loop10" successfully created.
  Can't open /dev/sdb1 exclusively.  Mounted filesystem?
  Can't open /dev/sdb1 exclusively.  Mounted filesystem?
Error: Process completed with exit code 5.
```

**Comment by easimon** [comment]
Thanks for reporting this. I have not found what locks the disk, but I think I found a way to work around it.
Could you re-test your build job with `easimon/maximize-build-space@fix/ubuntu-20.04` before I merge and tag it?

**Comment by tomyun** [comment]
Tested with the new branch and it worked great for me. Thanks!
